config/locales/en.yml

```diff
@@ -1,22 +1,22 @@
 # English strings go here for Rails i18n
 en:
+  global_popup_title: "Redmine Advanced Messenger"
+  
   issue_notifications_dialog_go_to_first_one: "Go to first one"
-  issue_notifications_dialog_text_for_current_user_1: "For"
-  issue_notifications_dialog_text_for_current_user_2: "there are"
-  issue_notifications_dialog_text_for_current_user_3: "no"
-  issue_notifications_dialog_text_for_current_user_4: "unread"
-  issue_notifications_dialog_text_for_current_user_5: "notifications"
+  issue_notifications_dialog_text_for_current_user_1: "For %{user}, there are"
+  issue_notifications_dialog_text_for_current_user_2: "<span style='color: red'> unread </span> notifications."
```

Comment:
I still think you could have used a singe entry here

```
"For %{user}, there are <span id="unread-notifications-count"></span> <span style='color: red'> unread </span> notifications."
```

I don't see where you replace `<span id="unread-notifications...></span>` whth "no" as in "no unread notifications, but probably is doable like this as well?

Reply:
I put the message instead: there are 0 unread notifications